asApp/src/save_restore.c

```diff
@@ -453,98 +453,99 @@ int manual_save(char *request_file, char *save_file, callbackFunc callbackFuncti
+void save_restoreSet_IncompleteSetsOk(int ok) { save_restoreIncompleteSetsOk = ok; }
+void save_restoreSet_DatedBackupFiles(int ok) { save_restoreDatedBackupFiles = ok; }
+static int save_restorePeriodicDatedBackups = 0;
 static int save_restoreDatedBackupPeriod;
-void save_restoreSet_periodicDatedBackups(int periodInMinutes) {
-	if (periodInMinutes>0) {
-		save_restorePeriodicDatedBackups = 1;
-		save_restoreDatedBackupPeriod = periodInMinutes*60;
-	} else {
-		save_restorePeriodicDatedBackups = 0;
-	}
+void save_restoreSet_periodicDatedBackups(int periodInMinutes)
+{
+    if (periodInMinutes > 0) {
+        save_restorePeriodicDatedBackups = 1;
+        save_restoreDatedBackupPeriod = periodInMinutes * 60;
+    } else {
+        save_restorePeriodicDatedBackups = 0;
+    }
 }
-void save_restoreSet_status_prefix(char *prefix) {strNcpy(status_prefix, prefix, 30);}
+void save_restoreSet_status_prefix(char *prefix) { strNcpy(status_prefix, prefix, 30); }
 #if SET_FILE_PERMISSIONS
-void save_restoreSet_FilePermissions(int permissions) {
-	file_permissions = (mode_t)permissions;
-	mustSetPermissions = 1;
-	printf("save_restore: File permissions set to 0%o\n", (unsigned int)file_permissions);
+void save_restoreSet_FilePermissions(int permissions)
+{
+    file_permissions = (mode_t)permissions;
+    mustSetPermissions = 1;
+    printf("save_restore: File permissions set to 0%o\n", (unsigned int)file_permissions);
 }
 #endif
-void save_restoreSet_RetrySeconds(int seconds) {
-	if (seconds >= 0) save_restoreRetrySeconds = seconds;
+void save_restoreSet_RetrySeconds(int seconds)
+{
+    if (seconds >= 0) save_restoreRetrySeconds = seconds;
 }
-void save_restoreSet_UseStatusPVs(int ok) {save_restoreUseStatusPVs = ok;}
-void save_restoreSet_CAReconnect(int ok) {save_restoreCAReconnect = ok;}
-void save_restoreSet_CallbackTimeout(int t) {
-	if ((t<0) || (t>=MIN_PERIOD)) {
-		save_restoreCallbackTimeout = t;
-	} else {
-		printf("save_restoreCallbackTimeout must be either negative (forever) or >= %d seconds\n", MIN_PERIOD);
-	}
+void save_restoreSet_UseStatusPVs(int ok) { save_restoreUseStatusPVs = ok; }
+void save_restoreSet_CAReconnect(int ok) { save_restoreCAReconnect = ok; }
+void save_restoreSet_CallbackTimeout(int t)
+{
+    if ((t < 0) || (t >= MIN_PERIOD)) {
+        save_restoreCallbackTimeout = t;
+    } else {
+        printf("save_restoreCallbackTimeout must be either negative (forever) or >= %d seconds\n", MIN_PERIOD);
+    }
 }
 
 /********************************* code *********************************/
 
 int isValid1stPVChar(char chr)
 {
-  return isalpha((int)chr) || isdigit((int)chr) || chr == '_' || chr == '-'
-	  || chr == '+' || chr == ':' || chr == '[' || chr == ']' || chr == '<'
-	  || chr == '>' || chr == ';';
+    return isalpha((int)chr) || isdigit((int)chr) || chr == '_' || chr == '-' || chr == '+' || chr == ':' || chr == '[' || chr == ']' || chr == '<' || chr == '>' || chr == ';';
```

Comment:
The previous line-wrapping here was better then putting everything on one line.

Reply:
This is the result of unlimited column, that I've set to avoid reformatting some `fprintf`s.
It seems that column limit of 120 characters does a good job of keeping long lines in check without excessively reformatting long messages.